packages/dids/tests/methods/did-ion.spec.ts

```diff
@@ -1,16 +1,16 @@
+import type { DidDocument } from '../../src/types/did-core.js';
 import type { Jwk } from '@web5/crypto';
+import type { PortableDid } from '../../src/types/portable-did.js';
 
+import chaiAsPromised from 'chai-as-promised';
```

Comment:
```suggestion
```

Reply:
Discussed offline, agreed to leave it until #639 is addressed.